### PR TITLE
Login Validation

### PR DESCRIPTION
**Pull request by aortizoj15**

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -20,26 +21,27 @@ type Values = {
...
-  const res = await fetch(`${SERVER_URL}/signin`, {
-    method: 'POST',
-    headers: {
-      'content-type': 'application/json'
-    },
-    body: JSON.stringify({
-      username: values.username,
-      password: values.password
+const Login: React.FC<Props> = () => {
+  const [invalidCredentials, setInvalidCredentials] = useState(false)
```
Comment (Wolfy64) [inline pages/login.tsx]: Your state logic is to display or not `<Alert />` I would rather name the state:
 `[isAlertVisible setIsAlertVisible] = useState(false)`

File: pages/login.tsx [hunk 2]
```diff
@@ -20,26 +21,27 @@ type Values = {
...
+      body: JSON.stringify({
+        username: values.username,
+        password: values.password
+      })
     })
-  })
-  const data = await res.json()
-  if (data.success) {
+    if (res.redirected) {
+      setInvalidCredentials(true)
```
Comment (Wolfy64) [inline pages/login.tsx]: if I read `setInvalidCredentials(true) ` I'm expecting you are doing some kind of logic but not to display a message.

File: helpers/loginUser.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const SERVER_URL = process.env.SERVER_URL
+
+export const loginUser = async (username: string, password: string) => {
```
Comment (songz) [inline helpers/loginUser.ts]: tests for this?

File: __tests__/loginUserTest.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { loginUser } from '../helpers/loginUser'
+describe('loginUser helper', () => {
+  test('Should return object', async () => {
+    window.fetch = jest.fn().mockReturnValue(
+      Promise.resolve({
+        redirected: false
+      })
+    )
+    expect(Promise.resolve(loginUser('user123', 'pass123'))).resolves.toBe({})
```
Comment (songz) [inline __tests__/loginUserTest.js]: you can also do this:

```
const user = await loginUser(....)
expect(user).toEqual({})
```

**Comment by songz** [comment]
Closes #33 - Login form validation

**Pull request merged**